easybuild/easyconfigs/i/IMB/IMB-4.1-foss-2016a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """
+The Intel® MPI Benchmarks perform a set of MPI performance measurements for point-to-point and global communication operations for a range of message sizes.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(name)s_%(version)s.tgz']
+source_urls = ['https://software.intel.com/sites/default/files/managed/a3/b2/']
+
+prebuildopts = 'export MPI_HOME=$EBROOTOPENMPI && cd src &&'
+buildopts = '-f make_mpich'
+
+patches = ['IMB-make_mpich.patch']
+
+parallel = 1
+
+files_to_copy = [
+(['src/%s' % binfile for binfile in ['IMB-RMA', 'IMB-EXT', 'IMB-NBC', 'IMB-MPI1']], 'bin'),
```

Comment:
@RvDijk existing IMB easyconfig have this, which makes more sense to me since then all `IMB-*` binaries are copied (even if some future updates adds a new one not listed here):

```python
files_to_copy = [(['src/IMB-*'], 'bin')]
```

so, I'd go with:

```python
files_to_copy = [(['src/IMB-*'], 'bin'), (['doc/IMB_Users_Guide.pdf'], 'doc')]
```